docs/tutorial_taskflow_api.rst

```diff
@@ -144,6 +144,22 @@ the dependencies as shown below.
     :end-before: [END main_flow]
 
 
+Multiple outputs inference
+--------------------------
+Tasks can also infer multiple outputs by using dict python typing.
+
+.. code-block:: python
+
+    @task
+    def identity_dict(x: int, y: int) -> Dict[str, int]:
+        return {"x": x, "y": y}
+
+By using the typing ``Dict`` for the function return type, the ``multiple_outputs`` parameter
+is automatically set to true.
+
+Not, If you manually set the ``multiple_outputs`` parameter the inference is disabled and
```

Comment:
```suggestion
Note, if you manually set the ``multiple_outputs`` parameter the inference is disabled and
```
Or did you have something else in your mind?

Reply:
nah def a typo